.travis.yml

```diff
@@ -3,18 +3,8 @@ sudo: required
 services:
   - docker
 
-env:
-    matrix:
-        - DOCKER_IMAGE=centos:6 CMAKE_URL=auto
-        - DOCKER_IMAGE=centos:7 CMAKE_URL=auto
-        - DOCKER_IMAGE=debian:7 CMAKE_URL=auto
-        - DOCKER_IMAGE=debian:8 CMAKE_URL=auto
-        - DOCKER_IMAGE=debian:8 CMAKE_URL=auto CC=clang CXX=clang++
-        - DOCKER_IMAGE=debian:stretch
-        - DOCKER_IMAGE=fedora:latest
-        - DOCKER_IMAGE=ubuntu:12.04 CMAKE_URL=auto
-        - DOCKER_IMAGE=ubuntu:latest CMAKE_URL=auto # LTS
-        - DOCKER_IMAGE=ubuntu:devel
-
 script:
-    - ./build/run.sh build
+  - docker build -t mozilla/lua_sandbox_extensions .
+  - |
+    docker run --rm mozilla/lua_sandbox_extensions
+    bash -c 'cd /root/lua_sandbox_extensions/release && ctest -V && ctest -V -C integration -j 10'
```

Comment:
The integration tests run the base tests too so you only need the last ctest command